Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -132,6 +132,53 @@ export async function findSigningIdentity(keychainPath: string) {
     }
 }
 
+/**
+ * Check if provisioning profile contains Cloud entitlement 
+ * @param provisioningProfilePath
+ * @returns {boolean} 
+ */
+export async function includesCloudEntitlement(provisioningProfilePath: string) {
+    //find the provisioning profile details
+    let provProfileDetails: string;
+    const getProvProfileDetailsCmd: ToolRunner = tl.tool(tl.which('security', true));
+    getProvProfileDetailsCmd.arg(['cms', '-D', '-i', provisioningProfilePath]);
+    getProvProfileDetailsCmd.on('stdout', function (data) {
+        if (data) {
+            if (provProfileDetails) {
+                provProfileDetails = provProfileDetails.concat(data.toString().trim().replace(/[,\n\r\f\v]/gm, ''));
+            } else {
+                provProfileDetails = data.toString().trim().replace(/[,\n\r\f\v]/gm, '');
+            }
+        }
+    });
+    
+    await getProvProfileDetailsCmd.exec();
+
+    let tmpPlist: string;
+    if (provProfileDetails) {
+        //write the provisioning profile to a plist
+        tmpPlist = '_xcodetasktmp.plist';
+        tl.writeFile(tmpPlist,provProfileDetails);
```

Comment:
Missing space after the comma. You can try Shift+Option+F in VS Code on Mac to handle the formatting for you.